plugin/evm/gossiper.go

```diff
@@ -259,7 +270,24 @@ func (n *pushGossiper) awaitEthTxGossip() {
 						"len(txs)", attempted,
 						"err", err,
 					)
+					continue
```

Comment:
I don't think we actually need to `continue` here. But this `warn` shouldn't be firing anyways.. So it seems fine.